src/test/run-pass/rfcs/rfc-2126-extern-absolute-paths/whitelisted.rs

```diff
@@ -11,6 +11,8 @@
 // run-pass
 // edition:2018
 
+extern crate core;
```

Comment:
We would want a dedicated test just for this I think a of copy this file with your change and one without.